sites/svelte-5-preview/src/routes/docs/content/01-api/03-snippets.md

```diff
@@ -265,3 +265,5 @@ In advanced scenarios, you may need to create a snippet programmatically. For th
 In Svelte 4, content can be passed to components using [slots](https://svelte.dev/docs/special-elements#slot). Snippets are more powerful and flexible, and as such slots are deprecated in Svelte 5.
 
 They continue to work, however, and you can mix and match snippets and slots in your components.
+
+When using custom elements, you should still use `<slot />` like before. In a future version, when Svelte removes its internal version of slots, it will leave those slot as-is, i.e. output a regular DOM tag instead of transforming it.
```

Comment:
```suggestion
When using custom elements, you should still use `<slot />` like before. In a future version, when Svelte removes its internal version of slots, it will leave those slots as-is, i.e. output a regular DOM tag instead of transforming it.
```